caddyhttp/httpserver/server.go

```diff
@@ -507,16 +507,35 @@ func (s *Server) Stop() error {
 // OnStartupComplete lists the sites served by this server
 // and any relevant information, assuming caddy.Quiet == false.
 func (s *Server) OnStartupComplete() {
-	if caddy.Quiet {
-		return
+	if !caddy.Quiet {
+		firstSite := s.sites[0]
+		scheme := "HTTP"
+		if firstSite.TLS.Enabled {
+			scheme = "HTTPS"
+		}
+		fmt.Println("")
+		fmt.Printf("Serving %s on port "+firstSite.Port()+" \n", scheme)
+		s.OutputSiteInfo("fmt")
+	}
+	// if caddy process log is going to stdout, printing to log
+	// here would duplicate to stdout so dont.  If -quiet ensure
+	// that Log is still output even if process log startup isnt. #2469
+	if caddy.LogDestination != "stdout" || caddy.Quiet {
+		s.OutputSiteInfo("log")
 	}
+}
+
+func (s *Server) OutputSiteInfo(LogType string) {
```

Comment:
This variable name should be `logType` -- and the function should be documented, since it is exported. Godoc is required here.

Reply:
I'll modify the param name.  Also the function doesnt need to be exported so I'll change.

Reply:
Cool. I do love that this change moves it into a separate function. :+1: